app/views/layouts/application.html.erb

```diff
@@ -14,17 +14,24 @@
     <script src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.0.1/dist/alpine.js" defer></script>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
 
-    <% if Rails.env == 'production' %>
-      <!-- Global site tag (gtag.js) - Google Analytics -->
-      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-162054776-1"></script>
+      <%# Always load GA, for consistency, but use different accounts for development vs. production %>
+      <script async src="https://www.googletagmanager.com/gtag/js?id=<%= google_analytics_id %>"></script>
       <script>
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         gtag('js', new Date());
 
-        gtag('config', 'UA-162054776-1');
+        gtag('config', '<%= google_analytics_id %>');
+
```

Comment:
You're going to need to take turbolinks into consideration:

https://michaelsoolee.com/google-analytics-rails-5/